integration/v3_watch_test.go

```diff
@@ -286,6 +288,8 @@ func TestV3WatchFromCurrentRevision(t *testing.T) {
 			t.Errorf("unexpected pb.WatchResponse is received %+v", nr)
 		}
 
+		<-ch
```

Comment:
Add a comment here? Something like "wait for the client to finish sending the keys before terminating the cluster".